agent/php_execute.h

```diff
@@ -64,6 +64,8 @@ extern void nr_framework_create_metric(TSRMLS_D);
  *           This is necessary to correctly instrument frameworks and libraries
  *           that are preloaded.
  */
+
 extern void nr_php_user_instrumentation_from_opcache(TSRMLS_D);
 
+#include "php_observer.h"
```

Comment:
I would suggest placing this at the top of the file after `#define PHP_EXECUTE_HDR` so it's more visible at a glance.